frameworks/g-cloud-12/questions/services/virtualisationThirdPartyProvider.yml

```diff
@@ -11,9 +11,8 @@ type: text
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter the provider name.
   - name: under_10_words
-    message: Your description must be no more than 10 words.
+    message: Provider name must be 10 words or fewer.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?